tsl/src/fdw/option.c

```diff
@@ -273,3 +283,58 @@ option_get_from_options_list_int(List *options, const char *optionname, int *val
 
 	return found;
 }
+
+List *
+option_extract_join_ref_table_list(const char *table_string)
+{
+	List *ref_table_oids = NIL;
+	List *ref_table_list;
+	ListCell *lc;
+
+	/* SplitIdentifierString scribbles on its input, so pstrdup first */
+	if (!SplitIdentifierString(pstrdup(table_string), ',', &ref_table_list))
+	{
+		/* syntax error in name list */
+		ereport(ERROR,
+				(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
+				 errmsg("parameter \"%s\" must be a list of reference table names",
```

Comment:
```suggestion
				 errmsg("parameter \"%s\" must be a comma-separated list of reference table names",
```

Reply:
@erimatnor Thanks for the review. The error messages are changed in the current version of the PR.